### PR TITLE
Planning considerations assembly added

### PR DESCRIPTION
**Pull request by apinnick**
No Jira or BZ number

This is an assembly created for these modules:
https://bugzilla.redhat.com/show_bug.cgi?id=1968352
https://bugzilla.redhat.com/show_bug.cgi?id=1968377

CP 4.6+

No QE required. Peer review required.

### CODE REVIEW

**Review by jeana-redhat — COMMENTED** [review]
Just one thing for forming an assembly - needs a toc line :)

File: migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[id="planning-considerations-3-4"]
+= Planning considerations
+include::modules/common-attributes.adoc[]
+:context: planning-considerations-3-4
+
```
[the remark this answers is not in the document]
Comment (apinnick) [reply migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc:5]: Thanks for the reminder! I forgot that. Will add now.

**Pull request merged**